tests/protractor/conf.js

```diff
@@ -1,61 +1,70 @@
-var _ = require('underscore'),
-    utils = require('./utils'),
-    spawn = require('child_process').spawn,
-    environment = require('./auth')(),
-    api;
+const _ = require('underscore'),
+      utils = require('./utils'),
+      spawn = require('child_process').spawn,
+      environment = require('./auth')(),
+      modules = [];
 
 _.templateSettings = {
   interpolate: /\{\{(.+?)\}\}/g
 };
 
-var login = function(browser) {
-  var loginUrlTemplate = _.template('http://{{apiHost}}:{{apiPort}}/{{dbName}}/login?redirect=');
-  var redirectUrl = encodeURIComponent('/' + environment.dbName  + '/_design/medic/_rewrite/#/messages?e2eTesting=true');
+const login = browser => {
+  const loginUrlTemplate = _.template('http://{{apiHost}}:{{apiPort}}/{{dbName}}/login?redirect=');
+  const redirectUrl = encodeURIComponent('/' + environment.dbName  + '/_design/medic/_rewrite/#/messages?e2eTesting=true');
   browser.driver.get(loginUrlTemplate(environment) + redirectUrl);
   browser.driver.findElement(by.name('user')).sendKeys(environment.user);
   browser.driver.findElement(by.name('password')).sendKeys(environment.pass);
   browser.driver.findElement(by.id('login')).click();
   // Login takes some time, so wait until it's done.
-  return browser.driver.wait(function() {
+  return browser.driver.wait(() => {
     return browser.driver.isElementPresent(by.css('body.bootstrapped'));
   }, 20 * 1000, 'Login should be complete within 20 seconds');
 };
 
-var startApi = function() {
-  return new Promise(function(resolve) {
-    var couchUrlTemplate = _.template('http://{{user}}:{{pass}}@{{couchHost}}:{{couchPort}}/{{dbName}}');
-    api = spawn('node', ['server.js'], {
-      cwd: 'api',
+const startNodeModule = (dir, startOutput) => {
+  return new Promise(resolve => {
+    const couchUrlTemplate = _.template('http://{{user}}:{{pass}}@{{couchHost}}:{{couchPort}}/{{dbName}}');
+    const module = spawn('node', ['server.js'], {
+      cwd: dir,
       env: {
         API_PORT: environment.apiPort,
         COUCH_URL: couchUrlTemplate(environment),
         COUCH_NODE_NAME: process.env.COUCH_NODE_NAME,
         PATH: process.env.PATH
       }
     });
-    api.stdout.on('data', function(data) {
-      if (data.toString().indexOf('Medic API listening on port') >= 0) {
```

Comment:
I was going to say that ES6 supports `.startsWith`, but this is checking `>= 0`. What happens if it's at index 0? This code really just means "it exists at all" right? so like `data.toString().includes('Medic API listening on port')`?